src/shaders/MaterialX/mx_screen.mx

```diff
@@ -14,5 +14,5 @@ shader SHADER_NAME(mx_screen)
     output TYPE out = TYPE_DEFAULT_OUT
   )
 {
-    out =  invert(multiply(invert(fg), invert(bg))); 
+    out = 1 - (1 - fg) * (1 - bg); 
```

Comment:
that didn't work, it interpreted the `*` as bold. haha. Let me try again:
I think this is equivalent to `1 - (1 - bg - fg + fg*bg)`
and thus to `fg + bg - fg*bg`